### PR TITLE
Bump cluster autoscaler to latest version

### PR DESCRIPTION
**Pull request by whites11**
### What this PR does / why we need it

We need newer cluster-autoscaler for private clusters to have proxy envs injected

### Checklist

- [ ] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: helm/default-apps-aws/values.yaml [hunk 1]
```diff
@@ -163,7 +163,7 @@ apps:
     namespace: kube-system
     # used by renovate
     # repo: giantswarm/cluster-autoscaler-app
-    version: 1.27.3-gs3
+    version: 1.27.3-gs7
```
Comment (AndiDog) [inline helm/default-apps-aws/values.yaml:166]: From looking into the changelog, isn't the latest version `...-gs8` (incl. proxy settings)?

Comment (whites11) [reply helm/default-apps-aws/values.yaml]: that was released afterwards, but I bumped it anyway

**Review by mnitchev — APPROVED** [review]

**Pull request merged**